From bf8f81bc254642110b009e802e479be1ea149423 Mon Sep 17 00:00:00 2001
From: Jim Harris <james.r.harris@intel.com>
Date: Mon, 1 Feb 2021 14:28:30 +0000
Subject: [PATCH 02/15] env_dpdk: add rte_ethdev dependency

As of DPDK main commit 682a6454 ("power: add ethdev
power management"), rte_power depends on rte_ethdev.
So add rte_ethdev library, but only on Linux since
that is the only place where we use rte_power.

Signed-off-by: Jim Harris <james.r.harris@intel.com>
Reviewed-on: https://review.spdk.io/gerrit/c/spdk/spdk/+/6210 (master)

(cherry picked from commit 710838103bb8ba6e9938dd6aa66c72c6ecb3e928)
Change-Id: Ibc3473d6ec75c14a8c0c5517616950017cb2ea96
Signed-off-by: Tomasz Zawadzki <tomasz.zawadzki@intel.com>
Reviewed-on: https://review.spdk.io/gerrit/c/spdk/spdk/+/6320
Reviewed-by: Jim Harris <james.r.harris@intel.com>
Reviewed-by: Ben Walker <benjamin.walker@intel.com>
Tested-by: SPDK CI Jenkins <sys_sgci@intel.com>
---
 lib/env_dpdk/env.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/env_dpdk/env.mk b/lib/env_dpdk/env.mk
index 6dae04e..f44f515 100644
--- a/lib/env_dpdk/env.mk
+++ b/lib/env_dpdk/env.mk
@@ -51,7 +51,7 @@ DPDK_INC := -I$(DPDK_INC_DIR)
 DPDK_LIB_LIST = rte_eal rte_mempool rte_ring rte_mbuf rte_pci rte_bus_pci rte_mempool_ring
 
 ifeq ($(OS),Linux)
-DPDK_LIB_LIST += rte_power
+DPDK_LIB_LIST += rte_power rte_ethdev
 endif
 
 # DPDK 20.05 eal dependency
-- 
1.8.3.1

